README.md

```diff
@@ -122,6 +122,15 @@ is a representation of the entity producing telemetry.
 |--------------------------|--------------------------|------------------------------------------------------------------------------|
 | otel.resource.attributes | OTEL_RESOURCE_ATTRIBUTES | Used to specify resource attributes in format: key1=val1,key2=val2,key3=val3 |
 
+#### Peer service name
+
+The [peer service name](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/span-general.md#general-remote-service-attributes) is
+the name of a remote service that is being connected to. It roughly corresponds to `service.name` in the [Resource](https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service) for the local service.
+
+| System property                     | Environment variable              | Purpose                                                                      |
+|------------------------------------|------------------------------------|------------------------------------------------------------------------------|
+| otel.endpoint.peer.service.mapping | OTEL_ENDPOINT_PEER_SERVICE_MAPPING | Used to specify a mapping from hostnames or IP addresses to peer services, as a comma separated list of host=name pairs. The peer service name will be added as an attribute to a span whose host or IP match the mapping. For example, 1.2.3.4=cats-service,dogs-abcdef123.serverlessapis.com=dogs-api |
```

Comment:
Should you say attribute key here as well?

Reply:
I'm not sure I understood this suggestion but tried something, let me know if it's what you were looking for.